src/main/java/org/kpmp/geneExpressionSummary/SCRNAGeneExpressionSummaryRepository.java

```diff
@@ -27,4 +27,6 @@
 	@Query(value = "SELECT COUNT(*) FROM sc_rnaseq scr WHERE scr.gene= :gene", nativeQuery = true)
 	long getCountByGene(@Param("gene") String gene);
 
+    @Query(value = "SELECT COUNT(*) FROM sc_rnaseq scr WHERE scr.gene= :geneSymbol AND scr.tissue_type= :tissueType", nativeQuery = true)

```

Comment:
I can't remember (and the card doesn't say) if we wanted these counts to be filtered by the gene searched for. If that is the case, then, ignore this comment.